app/models/sign.rb

```diff
@@ -103,6 +103,7 @@ def self.query_string_for_search(query)
   def self.http_conn
     Faraday.new(url: SIGN_URL) do |faraday|
       faraday.use FaradayMiddleware::FollowRedirects
+      faraday.options[:timeout] = 60
```

Comment:
Fetching the datadump from freelex takes quite a while and leads to faraday timeouts if you try to grab the whole thing at once. This is toootally a compromise but it appears to be the cleanest way to get all of the signs from that datadump we need at the same time for the sitemap.

Reply:
I thought so, just checking.... it could be that you made an optional argument to this that allows you to extend the timeout in specific scenarios but totally 🐘

Reply:
yeah we did discuss something like that but its tied into enough existing code that doing that in a nice way would be lootts of extra work.

Reply:
I think an optional argument could be added with almost no knock on effect to the codebase:
https://holgerjust.de/2016/detecting-default-arguments-in-ruby/

```def self.http_conn(timeout = 10)
    Faraday.new(url: SIGN_URL) do |faraday|
      faraday.use FaradayMiddleware::FollowRedirects
      faraday.options[:timeout] = timeout
    end
end```

Reply:
But 🐘 !